test/ml/test_ml.py

```diff
@@ -21,10 +22,26 @@ def setup_class(cls):
     def test_summarize_networks(self):
         dataframe = ml.summarize_networks([INPUT_DIR + 'test-data-s1/s1.txt', INPUT_DIR + 'test-data-s2/s2.txt', INPUT_DIR + 'test-data-s3/s3.txt',
                                            INPUT_DIR + 'test-data-longName/longName.txt', INPUT_DIR + 'test-data-longName2/longName2.txt',
-                                           INPUT_DIR + 'test-data-empty/empty.txt', INPUT_DIR + 'test-data-spaces/spaces.txt', INPUT_DIR + 'test-mixed-direction/mixed-direction.txt'])
+                                           INPUT_DIR + 'test-data-empty/empty.txt', INPUT_DIR + 'test-data-spaces/spaces.txt', INPUT_DIR + 'test-data-mixed-direction/mixed-direction.txt'])
         dataframe.to_csv(OUT_DIR + 'dataframe.csv')
         assert filecmp.cmp(OUT_DIR + 'dataframe.csv', EXPECT_DIR + 'expected-dataframe.csv', shallow=False)
 
+    def test_summarize_networks_less_values(self):
+        with pytest.raises(ValueError):
+            ml.summarize_networks([INPUT_DIR + 'test-data-unexpected-amount-of-values/less.txt'])
+
+    def test_summarize_networks_more_values(self):
+        with pytest.raises(ValueError):
+            ml.summarize_networks([INPUT_DIR + 'test-data-unexpected-amount-of-values/more.txt'])
+
+    def test_summarize_networks_empty_line(self):
+        with pytest.raises(ValueError):
```

Comment:
Why does this test throw the ValueError? The comment indicates that blank lines are okay and we should skip over them.

Reply:
now empty lines are not okay